### PR TITLE
mfl - Network accounts

### PR DESCRIPTION
**Pull request by matthewfl**

### CODE REVIEW

File: src/balanced.js [hunk 1]
```diff
@@ -384,7 +386,66 @@ var ba = {
...
+    }
+
+    if (!event.data) {
+        noDataError(ea.callback);
+        return;
+    }
+
+    var data = event.data;
+    data.token = event.data.token || event.data.code;
+    data.network = ea.network_name;
```
Comment (mjallday) [inline src/balanced.js]: provider according to the spec. i'm just about to push out a change to balanced to bring it inline.

Comment (matthewfl) [inline src/balanced.js]: alright, sounds good

File: src/balanced.js [hunk 2]
```diff
@@ -384,7 +386,66 @@ var ba = {
...
+        dialog = window.open(url, "", "top=" + (screen.height/2 - 275) + ", left=" + (screen.width/2 - 250) + ", width=500, height=550");
+    },
+    configure: function () {
+  /*jsonp(make_url('/jsonp/'+marketplace_href+'/configs', {}), function (json) {
+      ea.network = json;
+  });*/
+    }
+};
+
+addEvent(window, 'message', function (event) {
```
Comment (mjallday) [inline src/balanced.js:430]: i feel like we could skip around all these dirty global/static variables if we did something like `addEvent(window, 'message', onMessage.call(dialog, provider, callback))` (exact syntax may differ from advertised)

Comment (matthewfl) [inline src/balanced.js:430]: this is complicated, as what happens if you try to tokenize two different external accounts at the same time, they are both going to come in on post message, and afaik there is not a good way to tell the difference between them without increasing the complexity of the code.

**Pull request merged**